packages/reactotron-server/src/app/screens/Timeline/commands/ApiResponse.tsx

```diff
@@ -8,6 +8,11 @@ export class ApiResponse extends React.Component<Props> {
   render() {
     const { command } = this.props
 
-    return <div>{command.payload.request.url}</div>
+    return <div className="bg-content border flex flex-row justify-between p-6">
```

Comment:
Awesome! We need to work out a nice way to have the commands expandable similar to how they work in the app currently. I will see if I can make some way to make that happen in a shareable way.